inst/ms/aggreCAT.qmd

```diff
@@ -450,20 +477,22 @@ transparency of reporting, and journal reporting [@Hanea2021]. Within
 distributed as columns in the dataset, and for each claim `paper_id`,
 each participant `user_id` is assigned a logical `1` or `0` if they
 included that reasoning category in support of their Best estimate for
-that claim. See section ref(ReasonWAgg) for details on the `ReasonWAgg`
-aggregation method.
+that claim. See @sec-ReasoningWAgg for details on the
+`ReasonWAgg` aggregation method.
 
 #### Bayesian Prior Data {#sec-bayesian-supplementary-data}
 
-`BayPRIORsAgg()` uses Bayesian updating to update a prior probability of
-a claim replicating estimated from a predictive model [@Gould2021a]
-using an aggregate of the best estimates for all participants assessing
-a given claim $c$ [@Hanea2021]. The prior data is contained in
-`data_supp_priors` with each claim in column `paper_id` being
-assigned a prior probability of the claim replicating (on the logit
-scale) in column `prior_means`. (**TODO** should explain further about
-the mean / median of the distribution, ie internal workings of
-BayPRIORsAgg??).
+The method `BayPRIORsAgg` uses Bayesian updating to update a prior
+probability of a claim replicating estimated from a predictive model
+[@Gould2021a] using an aggregate of the best estimates for all
+participants assessing a given claim $c$ [@Hanea2021]. The prior data is
+contained in `data_supp_priors` with each claim in column `paper_id`
+being assigned a prior probability (on the logit scale) of the claim
+replicating in column `prior_means`. (**TODO** should explain further
```

Comment:
I don't think this is the place for going into the guys of `BayPRIORsAgg`, we already link off to the prereg and Hanea 2021.

As for the mean/median thing, we should be consistent to what we've coded up since that's what we submitted scores with for SCORE. Can't recall when or why the switch happened, but probably makes minimal dfference.

Reply:
Good points, agreed.